config/locales/client.zh_CN.yml

```diff
@@ -0,0 +1,88 @@
+en:
```

Comment:
change 'en' to 'zh'
Since that's the code for simplified Chinese

Reply:
I see. I removed the default placeholder locale for `zh` and update `zh_CN` translation.